components/Search.tsx

```diff
@@ -19,7 +19,10 @@ export default function Search({ searchBarWrapperStyles }: SearchProps) {
   });
 
   const onSubmit: SubmitHandler<FormValues> = ({ searchQuery }) => {
-    router.push({ query: { ...router.query, q: searchQuery } });
+    let trimmedQuery = searchQuery.trim();
+    if(trimmedQuery !== ''){ //Performs search only with non-empty strings
```

Comment:
Just run prettier to format the code now

Reply:
Added commit after formatting